src/clj/rems/poller/common.clj

```diff
@@ -22,7 +22,7 @@
     (try
       (doseq [e events]
         (try
-          (log/info name-kw "processing event" (:event/id e))
+          (log/debug name-kw "processing event" (:event/id e))
```

Comment:
I think this should be info level logging because it's pretty important to know that the async jobs are running. So far we don't have any monitoring for them, that would replace the need for this.

Consider also suppressing the logs in development mode for a solution to this.

Reply:
Created issue #1115